app/frontend/components/common/balance.module.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../variables.scss';
+
+.balanceRow {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.balanceTitle {
+  margin-bottom: 0;
+}
+
+.refreshButton {
+  padding: 10px 24px 10px 58px;
+  position: relative;
+
+  &::before {
+    content: '';
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    position: absolute;
+    left: 24px;
+    top: 0;
+    bottom: 0;
+    margin: auto;
+    width: 20px;
+    height: 20px;
+    background-image: url('../../../public/assets/refresh_button_symbol.svg');
+  }
+
+  &:hover::before {
+    background-image: url('../../../public/assets/refresh_button_symbol_active.svg');
+  }
+
+  @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-desktop-small) {
```

Comment:
I agree with introducing Sass. I looked around whether we could further shorten the `@media screen and` commands, since we're gonna see a lot of it from now on. Do you think solving this using mixins or something else would bring some value?

Reply:
I like the mixin idea :+1: Though I would go for it once we have this usage at few more places.